index.html

```diff
@@ -300,13 +300,11 @@ <h2>
     <p>
       If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I&#8217;d love to help with it 
     </p>
-    <form action="" class="form">
-      <input type="text" name="name" id="name" class="input-form">
-      <input type="email" name="email" id="email" class="input-form">
-      <textarea name="textarea" id="textarea" class="textarea" placeholder="write your message here"></textarea>
-      <a class="action" href="#">
-        <button class="button">get it touch</button>
-      </a>
+    <form action="https://formspree.io/f/%7Bform_id%7D" method="post" class="form">
```

Comment:
- [ ] Hi @zeff96, well done working on the contact section of your project, but according to the project requirement, we were asked to create a form-spree in such a way that any visitor can easily contact us when a message is submitted using the form-spree. but currently, the form-spree link is not working, kindly create an account on form-spree and use the id-link given to you in the `form action`

What the form is given when a message is submitted 👇 

![hashid](https://user-images.githubusercontent.com/78634841/212918345-64023daa-9c68-4237-b06d-bc79463260c3.png)